examples/allow_debugging_permissions/README.md

```diff
@@ -0,0 +1,28 @@
+# Integrate EKS cluster(s) Audit Logs with Lacework - Single Region
+
+This example creates all the required resources, as well as an IAM Role with a cross-account policy to
+provide Lacework read-only access to monitor the audit logs, and enables debugging permissions
+
+## Inputs
+
+| Name                          | Description                                                                                               | Type           |
+|-------------------------------|-----------------------------------------------------------------------------------------------------------|----------------|
+| `cloudwatch_regions`          | A list of regions, to allow Cloudwatch Logs to be streamed from                                           | `list(string)` |
+| `cluster_names`               | A set of cluster names, to integrate with. Defaults to [] if `no_cw_subscription_filter` is set to `true` | `set(string)`  |
+| `allow_debugging_permissions` | A boolean that can enable debugging permissions                                                           | `boolean`      |
+
+## Sample Code
+
+```terraform
+provider "lacework" {}
+
+module "aws_eks_audit_log" {
+  source = "../.."
```

Comment:
Update this to use source and version from the Terraform registry instead.

Look at other README's in the example folder.